src/po/Make_cyg.mak

```diff
@@ -65,14 +65,18 @@ PO_INPUTLIST = \
 
 first_time: $(PO_INPUTLIST) $(PO_VIM_INPUTLIST)
 	$(VIM) -u NONE --not-a-term -S tojavascript.vim $(LANGUAGE).pot $(PO_VIM_INPUTLIST)
-	$(XGETTEXT) --default-domain=$(LANGUAGE) \
+	$(XGETTEXT) --width=79 --package-name="Vim" --package-version="9.1" \
```

Comment:
I am fine with adding `--package-name="Vim"`, but please leave out `--package-version="9.1"` and `msgid-bugs-address`. I think bugs should better be adressed by the relevant maintainers, and package-version will just be another version number that needs to be addressed on a new release

Reply:
Yes indeed, about the version number I overlooked that, that's another place to remember to change when a new version comes out. I'll clean that up.

As for the "msgid-bugs-address" field, this is exactly what was done for developers. Here's what the documentation says:

> ‘--msgid-bugs-address=email@address’
> 
> Set the reporting address for msgid bugs. This is the email address or URL to which the translators shall report bugs in the untranslated strings: 

https://www.gnu.org/software/gettext/manual/gettext.html#index-_002d_002dmsgid_002dbugs_002daddress_002c-xgettext-option

That's why I made it auto-complete. I think that, probably, it is not necessary to clean it up.